docs/xml-config-file.rst

```diff
@@ -63,11 +67,12 @@ Library subsection, allows to define a types library for the upper type name. Th
 Finally, a default library for other types can be defined through the types_library subsection, similary to the library
 subsection.
 
+By default, any used type will be instantiate as :class:`GenericPubSubType`.
+
 Profiles
 --------
 
-The profiles section follows the format of Fast-RTPS XML Profiles. In this section participants, publisher and
-subscribers, with its configurations are defined.
+The profiles define participants, subscribers, publishers, etc, following the format used by **FastRTPS XML configuration files**, with its configuration.
```

Comment:
Wrong xml, subscriber profile cannot be inside participant profile.

Reply:
Fixed.